voxblox_fast/include/voxblox_fast/integrator/tsdf_integrator.h

```diff
@@ -166,11 +203,16 @@ class TsdfIntegrator {
       castRay(start_scaled, end_scaled, &global_voxel_indices);
       cast_ray_timer.Stop();
 
+      const float weight = getVoxelWeight(point_C);
```

Comment:
hmm ok, this can only stay here if we do const weight, could you add a check to make sure that if the config is not const weight it fails?